### PR TITLE
Added KubeCon+CloudNativeCon news article

### LINKED ISSUE #1215: [NG] News: KubeCon+CloudNativeCon: Service Mesh Battle Stories and Fixes
We're in the news! Using the contributing instructions for [adding news](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-news), please add the following article to the Layer5 Newsroom. Note: this issue is for the `layer5-ng` branch (site).

**Article:**
- Title: KubeCon+CloudNativeCon: Service Mesh Battle Stories and Fixes
- Link: https://thenewstack.io/kubeconcloudnativecon-service-mesh-battle-stories-and-fixes/

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by bartkalanski**
Signed-off-by: Bart Kalanski <bkalanski@gmail.com>

**Description**

This PR fixes #1215

**Notes for Reviewers**

Added the latest news story to the Layer5-ng branch

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/news/2020-12-02-kubecon+cloudnativecon-service-mesh-battle-stories-and-fixes/index.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+thumbnail: ./service-mesh-implementations.png
+eurl: https://thenewstack.io/kubeconcloudnativecon-service-mesh-battle-stories-and-fixes/
+published: true
+---
+
+import { NewsWrapper } from "../News.style.js";
+
+<NewsWrapper>
+
+As more organizations implement service meshes, they are finding what works and what needs more work, and they are creating new management practices around this knowledge. A few tried-and-tested best practices were detailed last month during KubeCon+CloudNativeCon.
```
Comment (kishore007k) [inline src/collections/news/2020-12-02-kubecon+cloudnativecon-service-mesh-battle-stories-and-fixes/index.mdx:15]: Add a little more content from the News. And some Images.

**Comment by Tanuj22** [comment]
@bartkalanski The build is failing probably due to the naming of the imports

**Comment by bartkalanski** [comment]
> @bartkalanski The build is failing probably due to the naming of the imports. Use camel case instead

Thanks for the tip!

**Review by kishore007k — APPROVED** [review]

**Review by Tanuj22 — APPROVED** [review]
Thanks for working on this @bartkalanski

**Pull request merged**